examples/Consumer/Program.cs

```diff
@@ -79,40 +79,53 @@ public static void Run_Consume(string brokerList, List<string> topics, Cancellat
             {
                 consumer.Subscribe(topics);
 
-                while (!cancellationToken.IsCancellationRequested)
+                try
                 {
-                    try
+                    while (true)
                     {
-                        var consumeResult = consumer.Consume(cancellationToken);
-
-                        if (consumeResult.IsPartitionEOF)
+                        try
                         {
-                            Console.WriteLine(
-                                $"Reached end of topic {consumeResult.Topic}, partition {consumeResult.Partition}.");
-
-                            continue;
+                            var consumeResult = consumer.Consume(cancellationToken);
+
+                            if (consumeResult.IsPartitionEOF)
+                            {
+                                Console.WriteLine(
+                                    $"Reached end of topic {consumeResult.Topic}, partition {consumeResult.Partition}.");
```

Comment:
print the offset too.
there is a consumeResult.TopicPartitionOffset for EOFs, right?